### ISSUE TITLE
No-unused-expression wrongly lints a comma-separated expression.

### ISSUE THREAD

**Issue by Aarilight** [palantir/tslint#2058]
### Bug Report

- __TSLint version__: `4.3.1`
- __TypeScript version__: `2.1.5`
- __Running TSLint via__: Visual Studio Code

#### TypeScript code being linted

```ts
let x: number, y: number;
x = 1, y = 2;
```

with `tslint.json` configuration:

```json
{
    "rules": {
		"no-unused-expression": true
	}
}
```

#### Actual behavior

Line 2 lints with `expected an assignment or function call`

#### Expected behavior

Line 2  contains two assignments, so it should not be linted.

**Comment by andy-hanson** [comment]
Why not just use `x = 1; y = 2;`?

**Comment by Aarilight** [comment]
Doesn't work on one line if statements.
